openff/toolkit/utils/openeye_wrapper.py

```diff
@@ -1488,7 +1543,8 @@ def from_smiles(
         from openeye import oechem
 
         oemol = oechem.OEGraphMol()
-        oechem.OESmilesToMol(oemol, smiles)
+        if not oechem.OESmilesToMol(oemol, smiles):
+            raise ParseError("Unable to parse the SMILES string")
```

Comment:
Nice. I've seen that OE uses this pattern a lot in their example code, so I'm glad that you know where to add it in here.